src/oatpp/network/Server.cpp

```diff
@@ -163,6 +163,14 @@ v_int32 Server::getStatus() {
   return m_status.load();
 }
 
+void Server::setConnectionProvider(const std::shared_ptr<ServerConnectionProvider> &connectionProvider) {
+  m_connectionProvider = connectionProvider;
```

Comment:
I think we should protect with the spin-lock.
Since server run loop is running in a different thread.

Reply:
Yes, just wanted to hear your opinion.